component/src/action_handler/actions/swap_claim.rs

```diff
@@ -87,16 +76,14 @@ impl ActionHandler for SwapClaim {
         let source = state.object_get("source").cloned().unwrap_or_default();
 
         state
-            .add_state_payload(StatePayload::Note {
-                source,
-                note: self.body.output_1.clone(),
-            })
+            .add_state_payload(StatePayload::RolledUp(
+                self.body.output_1_commitment.clone(),
+            ))
             .await;
         state
-            .add_state_payload(StatePayload::Note {
-                source,
-                note: self.body.output_2.clone(),
-            })
+            .add_state_payload(StatePayload::RolledUp(
+                self.body.output_2_commitment.clone(),
+            ))
```

Comment:
Now that we have support for bare note commitments, users don't have to expend any work scanning other users' swap outputs (which aren't directed at them anyways).